### ISSUE TITLE
includeExampleXml and includeExampleJson replacement?

### ISSUE THREAD

**Issue by nomisvai** [stoicflame/enunciate#502]
I am migrating from enunciate 1.26 to 2.6 and cannot find the options to remove the examples from the generated documentation, is there another way to achieve that with 2.6? If not, could it be added back?

**Comment by stoicflame** [comment]
No, that feature didn't make it into version 2. I'll look into adding it back for 2.7.

**Comment by stoicflame** [comment]
Fixed at 6cffd5f, but you have to disable examples for each provider module as needed, e.g.:

```xml
<enunciate>
  <modules>
    <jaxb disableExamples="true"/>
    <jackson1 disableExamples="true"/>
    <jackson disableExamples="true"/>
    <jaxrs disableExamples="true"/>
    <spring-web disableExamples="true"/>
```

**Comment by stoicflame** [comment]
[Enunciate 2.7 has been released.](https://github.com/stoicflame/enunciate/releases/tag/v2.7.0)
